app/api/resources/admin.py

```diff
@@ -21,6 +21,9 @@ class AssignNewUserAdmin(Resource):
     def post(cls):
         """
         Assigns a User as a new Admin.
+
+        An existing admin can use this endpoint to designate another user as an admin.
```

Comment:
grant instead of designate ?

Reply:
```suggestion
        An existing admin can use this endpoint to grant another user as an admin.
```

Reply:
I was about to commit and then squash, but I want to avoid erasing @daksha-aeer contribution. I guess this can be solved later. Improvement in english of Swagger docs :) @annabauza